PvPManager/src/main/resources/plugin.yml

```diff
@@ -36,7 +36,7 @@ commands:
       description: List all players with pvp enabled
       aliases: [plist, pvpl]
       permission: pvpmanager.list
-   tag:
+   pvptag:
       description: Shows time left until out of combat
       aliases: [ct, pt, pvptag]
```

Comment:
If the other commands also start with pvp it has sense to change this one, moreover if it is generating conflict with a minecraft command. Dont forget to remove pvptag as an alias